### PR TITLE
Update libreoffice from 6.4.5 to 7.0.0

### PR DESCRIPTION
**Pull request by saschpe**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

File: Casks/libreoffice.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask "libreoffice" do
-  version "6.4.5"
-  sha256 "8f8561085a93186ddb5d7d7b5f990e9effbb3305b1197768960bf6c688bef678"
+  version "7.0.0.3"
+  sha256 "74700ce918278b78464a4cd0e16d929e4af7b0447c42b400f61a2555818584e9"
 
   # documentfoundation.org/ was verified as official when first introduced to the cask
-  url "https://download.documentfoundation.org/libreoffice/stable/#{version}/mac/x86_64/LibreOffice_#{version}_MacOS_x86-64.dmg"
+  url "https://download.documentfoundation.org/libreoffice/stable/#{version.major_minor_patch}/mac/x86_64/LibreOffice_#{version}_MacOS_x86-64.dmg"
```
Comment (saschpe) [inline Casks/libreoffice.rb:6]: Thanks. Didn't knew this was a thing ;-)

Comment (vitorgalvao) [reply Casks/libreoffice.rb:6]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md#version-methods

**Comment by core-code** [comment]
does not look official to me 

```
LibreOffice 7.0 represents the bleeding edge in term of features for open source office suites, and as such is targeted at technology enthusiasts, early adopters and power users. The Document Foundation does not provide any technical support for users, although they can get help from other users on mailing lists and the Ask LibreOffice website: https://ask.libreoffice.org

For users whose main objective is personal productivity and therefore prefer a release that has undergone more testing and bug fixing over the new features, The Document Foundation maintains the LibreOffice 6.4 family, which includes some months of back-ported fixes. The current version is LibreOffice 6.4.5.
```

**Comment by core-code** [comment]
Screenshot 2020-08-05 at 17 19 36

**Comment by vitorgalvao** [comment]
@core-code [The appcast](https://download.documentfoundation.org/libreoffice/stable/) definitely led me to believe otherwise. It has “stable” in the path yet it’s presenting this versions. That means we’ll need to find another one.

**Comment by core-code** [comment]
on the other hand, i think we've been tracking their 'fresh' versions (in contrast to the 'still' versions) for a while now, and i think they have always been meant 'for enthusiasts'. 

if the 7.0.0 release shows up here in a few hours we'll have confirmation one or another way:

https://www.libreoffice.org/download/release-notes/#Fresh

**Comment by saschpe** [comment]
Fair enough.  It's been announced in [the Document Foundation's blog](https://blog.documentfoundation.org/blog/2020/08/05/announcement-of-libreoffice-7-0/)

**Comment by core-code** [comment]
it is now listed as 'released' here so i am fine with merging:

https://www.libreoffice.org/download/download/

**Comment by snaury** [comment]
Downloads for 7.0.0.3 give me 404 errors:

```
==> Upgrading libreoffice
==> Downloading https://download.documentfoundation.org/libreoffice/stable/7.0.0/mac/x86_64/LibreOffice_7.0.0.3_MacOS_x86-64.dmg
##O#- #                                                                       
curl: (22) The requested URL returned error: 404 
==> Purging files for version 7.0.0.3 of Cask libreoffice
```

The list of files seems to only contain 7.0.0, dated August 5:

https://download.documentfoundation.org/libreoffice/stable/7.0.0/mac/x86_64/

Could it be that 7.0.0.3 was some kind of prerelease build and now it should actually be 7.0.0?

**Comment by core-code** [comment]
@snaury thanks i believe this was fixed in https://github.com/Homebrew/homebrew-cask/pull/87159

**Pull request merged**